### PR TITLE
Initiate taskboard documentation

### PR DESCRIPTION
**Pull request by nterray**
Taskboard should be referenced in the user documentation.

### CODE REVIEW

File: languages/en/user-guide/agile-dashboard.rst [hunk 1]
```diff
@@ -171,24 +171,70 @@ To configure a velocity chart please refer to (:ref:`velocity_administration`) p
...
+    +---------------------------------------------------------------------------------------+----------+-----------+
+    | Display initial effort                                                                |          | ✓         |
+    +---------------------------------------------------------------------------------------+----------+-----------+
+    | Display number of cards in a column                                                   |          | ✓         |
+    +---------------------------------------------------------------------------------------+----------+-----------+
+    | Display in fullscreen                                                                 | ✓        | ✓         |
+    +---------------------------------------------------------------------------------------+----------+-----------+
+    | Change status of a card with drag 'n drop                                             | ✓        | ✓         |
+    +---------------------------------------------------------------------------------------+----------+-----------+
+    | Reorder cards with drag 'n drop                                                       | ✓        | ✓         |
```
Comment (vaceletm) [inline languages/en/user-guide/agile-dashboard.rst]: AFAICT it's not possible with cardwall

File: languages/en/user-guide/agile-dashboard.rst [hunk 2]
```diff
@@ -171,24 +171,70 @@ To configure a velocity chart please refer to (:ref:`velocity_administration`) p
...
+    +---------------------------------------------------------------------------------------+----------+-----------+
+    | Sticky (always visible) headers (both columns and swimlanes)                          |          | ✓         |
+    +---------------------------------------------------------------------------------------+----------+-----------+
+    | Hide closed items                                                                     |          | ✓         |
+    +---------------------------------------------------------------------------------------+----------+-----------+
+    | Expand/Collapse swimlanes                                                             |          | ✓         |
+    +---------------------------------------------------------------------------------------+----------+-----------+
+    | Expand/Collapse columns                                                               |          | ✓         |
+    +---------------------------------------------------------------------------------------+----------+-----------+
+    | Hide avatars on cards                                                                 | ✓        |           |
```
Comment (vaceletm) [inline languages/en/user-guide/agile-dashboard.rst]: I would not advertise that if you don't mind ;)

File: languages/en/user-guide/agile-dashboard.rst (hunk 1 above)
Comment (nterray) [reply languages/en/user-guide/agile-dashboard.rst]: Right! Bad copy paste 😕

File: languages/en/user-guide/agile-dashboard.rst (hunk 2 above)
Comment (nterray) [reply languages/en/user-guide/agile-dashboard.rst]: Ok 🙂

**Pull request merged**